manifest/types.go

```diff
@@ -4,17 +4,22 @@ import "github.com/forta-network/forta-core-go/utils"
 
 // AgentManifest represents the dev-provided properties of an agent
 type AgentManifest struct {
-	From           *string                       `json:"from"`
-	Name           *string                       `json:"name"`
-	AgentID        *string                       `json:"agentId"`
-	AgentIDHash    *string                       `json:"agentIdHash"`
-	Version        *string                       `json:"version"`
-	Timestamp      *string                       `json:"timestamp"`
-	ImageReference *string                       `json:"imageReference"`
-	Repository     *string                       `json:"repository"`
-	Documentation  *string                       `json:"documentation"`
-	ChainIDs       []int64                       `json:"chainIds"`
-	ChainSettings  map[string]AgentChainSettings `json:"chainSettings"`
+	From            *string                       `json:"from"`
+	Name            *string                       `json:"name"`
+	DisplayName     *string                       `json:"displayName"`
```

Comment:
displayName is only needed in the CLI (so we can remove from here) i.e. the CLI will use `displayName` if provided in package.json instead of `name` when creating the manifest for publishing